### ISSUE TITLE
Permissions required to edit the advance options of groups claim for application

### ISSUE THREAD

**Issue by yaegashi** [MicrosoftDocs/entra-docs#204]
I wanted to set up group filtering for a specific enterprise application on the Azure Portal. However, it seems that directory administrator privileges are required to edit the advanced options of the groups claim. Is the owner of the enterprise application unable to edit these options?

Could you please explain the permissions required to edit the group claim options for applications in detail? Additionally, I’m curious about how to configure it using the Microsoft Graph API.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 5f6626e5-bcb0-9abe-557b-a9cdb33cb3a6
* Version Independent ID: 147a86d5-1eec-123b-b757-efb29dbe5681
* Content: [Configure group claims for applications by using Microsoft Entra ID - Microsoft Entra ID](https://learn.microsoft.com/en-us/entra/identity/hybrid/connect/how-to-connect-fed-group-claims#group-filtering)
* Content Source: [docs/identity/hybrid/connect/how-to-connect-fed-group-claims.md](https://github.com/MicrosoftDocs/entra-docs/blob/main/docs/identity/hybrid/connect/how-to-connect-fed-group-claims.md)
* Service: **active-directory**
* Sub-service: **hybrid**
* GitHub Login: @billmath
* Microsoft Alias: **billmath**

**Comment by yaegashi** [comment]
Non-administrative users are unable to access advanced options for the groups claim by clicking on the "groups" row in the Attributes & Claims page.  And the message "Failed to load custom claims provider" is displayed, which may indicate a permission error.

![image](https://github.com/MicrosoftDocs/entra-docs/assets/106724/99dd1409-ce68-4782-82b1-5ff2158cf7ea)

**Comment by SaibabaBalapur-MSFT** [comment]
@yaegashi 
Thanks for your feedback! We will investigate and update as appropriate.

**Comment by SaibabaBalapur-MSFT** [comment]
@yaegashi 
I'm going to assign this to the document author so they can take a look at it accordingly.

@billmath 
Please review it.

**Comment by billmath** [comment]
Checking with engineering

**Comment by billmath** [comment]
Group claim configuration is part of the Application object, so I assume you would be able to update it with Application.ReadWrite permission.  However, if you have added a transform or filter to that group claim then that is part of the policy - so it would be Policy.ReadWrite.ApplicationConfiguration.  If that still doesn't work, you probably would need to open a ticket with CSS, since this may require more in-depth looking at the policy.  Thanks!

#please-close
